Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM debian:stretch-slim
```

Comment:
will we support Ubuntu in the future?

Reply:
No

Reply:
alright. perhaps I misunderstood what were these images meant for :) AFAIK, at least the shop stack runs on Ubuntu 16.04.

Reply:
Is there any reason to have an additional Docker image which does the same, but is just based on a different OS?

Reply:
These images are just supposed to use them for running Ansible itself. Either locally or for automated tests

Reply:
Also a valid use-case. So I'd suggest we merge it now and simply add another Dockerfile as soon as we need it.